app/models/qa/linked_data/config/context_map.rb

```diff
@@ -24,15 +24,15 @@ class ContextMap
         #       {
         #         "property_label_i18n": "qa.linked_data.authority.locgenres_ld4l_cache.authoritative_label",
         #         "property_label_default": "Authoritative Label",
-        #         "lpath": "madsrdf:authoritativeLabel",
+        #         "ldpath": "madsrdf:authoritativeLabel",
```

Comment:
Change all occurrences of lpath to ldpath, which is the actual name of the query language.